cmd/get.go

```diff
@@ -22,6 +22,7 @@ import (
 
 // MakeGet creates the Get command to download software
 func MakeGet() *cobra.Command {
+	var validToolOptions []string = []string{"faas-cli", "helm", "inletsctl", "k3d", "k3sup", "kind", "kubectl", "kubectx", "kubeseal"}
```

Comment:
I don't think that this list is required, because you have it in `tools := get.MakeTools()`